client/components/tinymce/plugins/mentions/suggestion.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const Suggestion = React.createClass( {
+	displayName: 'Suggestion',
```

Comment:
If a component class is assigned to a variable (or as a named class), you don't need to manually define `displayName` as this is inferred from its assigned name.

See also: http://babeljs.io/docs/plugins/transform-react-display-name/